Jenkinsfile_CNP

```diff
@@ -34,7 +34,8 @@ def secrets = [
         secret('ccd-befta-citizen-2-pwd', 'CCD_BEFTA_CITIZEN_2_PWD'),
         secret('ccd-befta-citizen-3-pwd', 'CCD_BEFTA_CITIZEN_3_PWD'),
         secret('ccd-befta-solicitor-3-pwd', 'CCD_BEFTA_SOLICITOR_3_PWD'),
-        secret('ccd-befta-master-caseworker-pwd', 'CCD_BEFTA_MASTER_CASEWORKER_PWD')
+        secret('ccd-befta-master-caseworker-pwd', 'CCD_BEFTA_MASTER_CASEWORKER_PWD'),
+        secret('TEST-DATA-RELOAD-FREQUENCY', 'TEST_DATA_RELOAD_FREQUENCY')
```

Comment:
Pls don't pick this from secrets. We really don't need to. An env var can already do it.